specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -3833,6 +3833,174 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/sqlPools/{sqlPoolName}/transparentDataEncryption/{transparentDataEncryptionName}/operationResults": {
+      "get": {
+        "tags": [
+          "SqlPools, TransparentDataEncryption"
+        ],
+        "operationId": "TransparentDataEncryptionActivities_ListByConfiguration",
```

Comment:
>TransparentDataEncryptionActivities_ListByConfiguration [](start = 24, length = 55)

SqlPoolTransparentDataEncryptionActivities_ListByConfiguration

Reply:
Not supporting this for now. I will send and email to the concerned parties and include you in it.